### ISSUE TITLE
Example issue title

### ISSUE THREAD

**Issue by mateogianolio** [mateogianolio/openblog#1]
Example issue body, rendered with [marked](https://github.com/chjj/marked).
1. List item
2. List item
3. List item

```
Code block
```

```javascript
{
  const a = 2;
  const b = 2;
  return a * b;
}
```

**Comment by mateogianolio** [comment]
Test comment

**Comment by mateogianolio** [comment]
Another test comment

**Comment by papercompute** [comment]
Is it possible to add comments (use API) on top page without jumping to github internal page?

**Comment by mateogianolio** [comment]
Yes, if you add user authentication, either via username/password login form or webflow, anything is possible :)

**Comment by dineshdharme** [comment]
Nice project mate!! Really liked it.

**Comment by mateogianolio** [comment]
Thanks! I appreciate it :)

**Comment by aviaryan** [comment]
Great idea @mateogianolio  .. I was thinking of a way to make a github powered blog without exposing users to terminal & code and here it is.

**Comment by mateogianolio** [comment]
I'm glad you like it.

**Comment by merqurio** [comment]
Great idea ! A nice way to use github as a cms ! :grin:

**Comment by mateogianolio** [comment]
Multi-line comment.

Test code:

```javascript
var test = 'yes';
```

**Comment by Amritanshu02** [comment]
Hello! I would like to work on this issue , could you suggest me where to start?
